.github/workflows/tests-backoffice-v3.yml

```diff
@@ -0,0 +1,98 @@
+on: workflow_call
+
+env:
+  region: europe-west1
+  tests_docker_image: europe-west1-docker.pkg.dev/passculture-infra-prod/pass-culture-artifact-registry/pcapi-tests:${{ github.sha }}
+
+defaults:
+  run:
+    working-directory: api
+
+jobs:
+  check-api-folder-changes:
+    name: "Check if folder changed"
+    uses: ./.github/workflows/check-folder-changes.yml
+    with:
+      folder: api
+
+  build-tests-docker-image:
+    name: "Build tests docker image"
+    uses: ./.github/workflows/build-and-push-docker-images.yml
+    needs: check-api-folder-changes
+    if: needs.check-api-folder-changes.outputs.folder_changed == 'true' || github.ref == 'refs/heads/master'
+    with:
+      tag: ${{ github.sha }}
+      tests: true
+    secrets: inherit
+
+  tests-api:
+    name: "Tests"
+    env:
+      RUN_ENV: tests
+      DATABASE_URL_TEST: postgresql://pytest:pytest@postgres:5432/pass_culture
+      REDIS_URL: redis://redis:6379
+    runs-on: ubuntu-latest
+    needs:
+      - check-api-folder-changes
+      - build-tests-docker-image
+    if: ${{ needs.check-api-folder-changes.outputs.folder_changed == 'true' }}
+    permissions:
+      id-token: write
+      contents: read
+    services:
+      redis:
+        image: redis:7-alpine
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      postgres:
+        image: postgis/postgis:12-3.3-alpine
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        env:
+          POSTGRES_USER: pytest
+          POSTGRES_PASSWORD: pytest
+          POSTGRES_DB: pass_culture
+    steps:
+      - uses: actions/checkout@v3
+      - id: openid-auth
+        name: "OpenID Connect Authentication"
+        uses: 'google-github-actions/auth@v0'
+        with:
+          create_credentials_file: false
+          token_format: 'access_token'
+          workload_identity_provider: ${{ secrets.INFRA_PROD_GCP_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.ARTIFACT_REGISTRY_SERVICE_ACCOUNT }}
+      - uses: docker/login-action@v2
+        with:
+          registry: '${{ env.region }}-docker.pkg.dev'
+          username: oauth2accesstoken
+          password: ${{ steps.openid-auth.outputs.access_token }}
+      - run: docker pull ${{ env.tests_docker_image }}
+      - name: Running backoffice tests
+        uses: addnab/docker-run-action@v3
+        with:
+          image: ${{ env.tests_docker_image }}
+          shell: bash
+          options: -e RUN_ENV -e DATABASE_URL_TEST -e REDIS_URL
+          run: |
+            mkdir -p test-results
+            pytest --durations=10 --junitxml=test-results/junit.xml -m backoffice_v3
```

Comment:
Un peu pareil que pour circleci, mais en github action ça se fait avec matrix je pense : 
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs
Tu réutilise le même jobs mais il s'execute 2 fois avec 2 valeurs différentes : backoffice_v3 et le cas normal, à voir comment le formaliser